### PR TITLE
Dataset tensor-conv-mobilenets-v1-0.75-160 added

### PR DESCRIPTION
**Pull request by Chunosov**
Issue https://github.com/ctuning/ck-nntest/issues/19

### CODE REVIEW

File: dataset/tensor-conv-mobilenets-v1-0.75-160/data.csv [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+MobileNet_0.75_160,20,20,192,1,192,1,0,,Conv2d_5
+MobileNet_0.75_160,10,10,192,1,384,1,0,,Conv2d_6
+MobileNet_0.75_160,10,10,384,1,384,1,0,,Conv2d_7
+MobileNet_0.75_160,10,10,384,1,384,1,0,,Conv2d_8
+MobileNet_0.75_160,10,10,384,1,384,1,0,,Conv2d_9
+MobileNet_0.75_160,10,10,384,1,384,1,0,,Conv2d_10
+MobileNet_0.75_160,10,10,384,1,384,1,0,,Conv2d_11
+MobileNet_0.75_160,5,5,384,1,768,1,0,,Conv2d_12
+MobileNet_0.75_160,5,5,768,1,768,1,0,,Conv2d_13
+MobileNet_0.75_160,1,1,1024,1,1001,1,0,,Logits_Conv2d_1c_1x1
```
Comment (psyhtest) [inline dataset/tensor-conv-mobilenets-v1-0.75-160/data.csv]: Shouldn't it be:
```
MobileNet_0.75_160,1,1,1024,1,1001,1,0,,Logits_Conv2d_1c_1x1
```
as in https://github.com/ctuning/ck-nntest/blob/master/dataset/tensor-conv1x1-mobilenets/data.csv#L93

Comment (psyhtest) [reply dataset/tensor-conv-mobilenets-v1-0.75-160/data.csv]: Let's also compare with all the 1x1 convolutions from https://github.com/ctuning/ck-nntest/blob/master/dataset/tensor-conv1x1-mobilenets/data.csv#L94 (sorted for convenience):
```
MobileNet_0.75_160,80,80,24,1,48,1,0,,Conv2d_1
MobileNet_0.75_160,40,40,48,1,96,1,0,,Conv2d_2
MobileNet_0.75_160,40,40,96,1,96,1,0,,Conv2d_3
MobileNet_0.75_160,20,20,96,1,192,1,0,,Conv2d_4
MobileNet_0.75_160,20,20,192,1,192,1,0,,Conv2d_5
MobileNet_0.75_160,10,10,192,1,384,1,0,,Conv2d_6
MobileNet_0.75_160,10,10,384,1,384,1,0,,Conv2d_7
MobileNet_0.75_160,10,10,384,1,384,1,0,,Conv2d_8
MobileNet_0.75_160,10,10,384,1,384,1,0,,Conv2d_9
MobileNet_0.75_160,10,10,384,1,384,1,0,,Conv2d_10
MobileNet_0.75_160,10,10,384,1,384,1,0,,Conv2d_11
MobileNet_0.75_160,5,5,384,1,768,1,0,,Conv2d_12
MobileNet_0.75_160,5,5,768,1,768,1,0,,Conv2d_13
```

Comment (psyhtest) [reply dataset/tensor-conv-mobilenets-v1-0.75-160/data.csv]: Everything's else appears to be proper.

Comment (Chunosov) [reply dataset/tensor-conv-mobilenets-v1-0.75-160/data.csv]: Sure, this is an error. The input of the layer should be the same as the output of the previous. Fixed.

**Pull request merged**